Makefile

```
@@ -4,7 +4,7 @@
 PWD = $(shell pwd)
 UID = $(shell id -u)
 GID = $(shell id -g)
-NPM_VERSION = $(shell python -m scripts.npm_version)
+NPM_VERSION = $(shell python -m scripts.generate_semver)
```

We could also do this

```suggestion
NPM_VERSION = $(shell poetry run scripts/generate_semver.py)
```